docs/mkdocs/docs/problems.md

```diff
@@ -0,0 +1,198 @@
+ERROR: /pub/svc/commerce/gobilda:gobilda: Error: ['product_value'] on the line: [return _run_module_code(code, init_globals, run_name,]
+ERROR: /pub/svc/commerce/gobilda:gobilda:
+ERROR: /pub/svc/commerce/gobilda:gobilda: 'product_value'
+INFO:  The following quotes are received:
+INFO:           /pub/svc/commerce/gobilda:gobilda: No quote received
+INFO:  DONE: SupplyQuote: this: 42.76s
+Aborted.
+```
+
+1. Error printed twice.
+2. Pretty print dictionary response as JSON.
+3. Make this error supported and display it in user friendly way.
+
+## 429 Resource has been exhausted (e.g. check quota).
+
+```
+    When I run "pc inspect 'generated-case'"                                                                               # features/steps/when.py:63 172.984s
+    Then the command should exit with a status code of "0"                                                                 # features/steps/partcad-cli/commands/version.py:50 0.001s
+      Traceback (most recent call last):
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/behave/model.py", line 1329, in run
+          match.run(runner.context)
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/behave/matchers.py", line 98, in run
+          self.func(context, *args, **kwargs)
+        File "/workspaces/partcad.worktrees/devel/features/steps/partcad-cli/commands/version.py", line 52, in step_impl
+          assert context.result.returncode == int(exit_code)
+      AssertionError
+
+      Captured logging:
+      DEBUG:root:Created temporary workspace: /tmp/sandbox/behave/partcad-cli-am2g4rb6
+      INFO:root:Created temporary home: /tmp/sandbox/home/partcad-cli-j5iagz3t
+      DEBUG:root:Running command: mkdir -pv /tmp/sandbox/home/partcad-cli-j5iagz3t/.partcad
+      cat <<EOF > /tmp/sandbox/home/partcad-cli-j5iagz3t/.partcad/config.yaml
+      # https://aistudio.google.com/apikey
+      googleApiKey: AIzaSyAA8eKtTSTt2nJzRHC-nrvKI5kW35ZgNV8
+      # YOLO settings to speed up the process
```

Comment:
Alex, Alex... Well I guess I need to add a pre-commit hook to avoid this in future.

P. S. It was free tier key which I just removed, but yes relying on keys in CI would have risks of leaking values in logs & etc.